examples/cloud-router-2-azure-connection/README.md

```diff
@@ -350,3 +350,339 @@ module "cloud_router_azure_connection" {
 | <a name="output_azurerm_express_route_circuit"></a> [azurerm\_express\_route\_circuit](#output\_azurerm\_express\_route\_circuit) | n/a |
 | <a name="output_azurerm_resource_group_id"></a> [azurerm\_resource\_group\_id](#output\_azurerm\_resource\_group\_id) | n/a |
 <!-- END_TF_DOCS -->
+
```

Comment:
The template is adding an additional TF_DOCS section. It should replace the existing one.

Reply:
Addressed with [commit](https://github.com/equinix/terraform-equinix-fabric/pull/102/commits/bd5bdbcc170f7547e6da899b1d44d6f5bc28720f)